requirements.txt

```diff
@@ -15,9 +15,6 @@ greenlet==0.4.15 ; python_version == '3.7'
 greenlet==0.4.17 ; python_version > '3.7'
 html2text==2018.1.9
 idna==2.6
-Jinja2==2.10.1; python_version < '3.8'
-# bullseye version, focal patched 2.10
-Jinja2==2.11.2; python_version >= '3.8'
```

Comment:
all the changes in dependencies should be reverted. Dropping this dependency is unnecessary, and will needlessly complicate our task in other places, by having to handle situations where people do not have jinja2 installed.

Reply:
So keep jinja as a hard dep and its other usages, only drop it from mail?

Reply:
I dropped all dependency changes.